packages/microcosm-www-next/src/layouts/index.js

```diff
@@ -2,28 +2,37 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import Link from 'gatsby-link'
 import Helmet from 'react-helmet'
+import VigetLogo from '../components/viget-logo'
 
 import '../stylesheets/app.scss'
 
-const Header = () => (
-  <div className='header'>
+const Navigation = () => (
+  <nav className="navigation">
     <div className='wrapper'>
-      <h1>
-        <Link
-          to="/"
-          style={{
-            color: 'black',
-            textDecoration: 'none',
-          }}
-        >
-          Microcosm
-        </Link>
-      </h1>
+      <Link to="/">
+        <img src="./static/images/logo.png" alt="Microcosm" />
```

Comment:
should I include `.screenreader-only` text for Microcosm, or is the `alt` enough?

Reply:
Alt should be enough. cc @ten1seven right?

Reply:
Alt should be enough, though I am not sure for SEO if not having a title to the page is ok?

Reply:
The `alt` attribute is enough for accessibility but @leobauza is correct, not having an `<h1>` is not good for overall accessibility or SEO because it hurts document structure.

Reply:
okay noted, I will wrap this in an `<h1>`